src/core/core.datasetController.js

```diff
@@ -849,14 +850,19 @@ export default class DatasetController {
 		const elementOptions = me.datasetElementOptions;
 		const values = {};
 		const context = me._getContext(undefined, active);
+		const defaultColors = {
+			backgroundColor: defaults.color,
+			borderColor: defaults.color,
+		};
 		let i, ilen, key, readKey, value;
 
 		for (i = 0, ilen = elementOptions.length; i < ilen; ++i) {
 			key = elementOptions[i];
 			readKey = active ? 'hover' + key.charAt(0).toUpperCase() + key.slice(1) : key;
 			value = resolve([
 				datasetOpts[readKey],
-				options[readKey]
+				options[readKey],
+				defaultColors[readKey],
```

Comment:
Could use something like `readKey.indexOf('Color') > -1 && defaults.color` to support possible other colors.
Neither one is a proper solution though :)

Reply:
That was actually my first pass, but it breaks the default hover colours